### ISSUE TITLE
Confused about data augmentation

### ISSUE THREAD

**Issue by adaxidedakaonang** [fcdl94/MiB#77]
In Subset.py, I saw you performed the normalization to both input and label. I'm confused about why using normalization to the label? 
```
train_transform = transform.Compose([
        transform.RandomResizedCrop(opts.crop_size, (0.5, 2.0)),
        transform.RandomHorizontalFlip(),
        transform.ToTensor(),
        transform.Normalize(mean=[0.485, 0.456, 0.406],
                            std=[0.229, 0.224, 0.225]),
    ])
sample, target = self.transform(sample, target)
```
From the pytorch offical document about cross entropy loss function, we know that the label is only used as index.

**Comment by fcdl94** [comment]
Normalization is not applied to the labels but only to the images. Please check out the transforms in https://github.com/fcdl94/MiB/blob/master/dataset/transform.py.
